stdlib/grpc/src/main/java/org/ballerinalang/net/grpc/builder/BallerinaFileBuilder.java

```diff
@@ -178,13 +181,15 @@ private void computeSourceContent(byte[] descriptor, String mode) throws CodeBui
             boolean needStubFile = serviceDescriptotList.size() != 1;
             boolean hasEmptyMessage = false;
             boolean enableEp = true;
+            boolean hasStreamingProxy = false;
+            boolean hasUnaryProxy = false;
             for (DescriptorProtos.ServiceDescriptorProto serviceDescriptor : serviceDescriptotList) {
                 ServiceStub.Builder serviceStubBuilder = ServiceStub.newBuilder(serviceDescriptor.getName());
                 ServiceFile.Builder sampleServiceBuilder = ServiceFile.newBuilder(serviceDescriptor.getName());
                 List<DescriptorProtos.MethodDescriptorProto> methodList = serviceDescriptor.getMethodList();
                 boolean isUnaryContains = false;
-                stubFileObject.setMessageMap(messageList.stream().collect(Collectors.toMap(Message::getMessageName,
-                        message -> message)));
+                boolean proxyAvailable = false;
```

Comment:
Do we need to recreate this flag every iteration? Can't we initialize at the beginning and reassign to false in here?

Reply:
If the .proto has multiple service definitions, this flag checks if each of these services has a proxy resource or not. Hence, the flag is recreated in each iteration for each service. 

Another approach i can follow is remove the flag all together, and replace line 210, with serviceStubBuilder.enableProxy()? Shall I make this change or keep it as it is?